src/Auth0.php

```diff
@@ -732,4 +787,20 @@ public function setStore(StoreInterface $store)
         $this->store = $store;
         return $this;
     }
+
+    /**
+     * @param integer $length
+     *
+     * @return string
+     */
+    public static function getNonce(int $length = 16) : string
```

Comment:
Out of curiosity, this should be pretty similar to the state generation. Are you duplicating that logic or can you re-use that one instead?

Reply:
I'll re-use, yes. I need to change how state is handled to match more closely with nonce in another PR